engine/apps/slack/scenarios/paging.py

```diff
@@ -507,7 +507,7 @@ def _get_organization_select(
 ) -> Block.Input:
     organizations_options: typing.List[CompositionObjects.Option] = []
     initial_option_idx = 0
-    for idx, org in enumerate(organizations):
+    for idx, org in enumerate(list(organizations)):
```

Comment:
I think this is because the type annotation `organizations: "RelatedManager['Organization']"` is not correct, looks like it should be `organizations: QuerySet["Organization"]`. Probably the wrong type annotation is introduced in the return type of `_get_available_organizations` methods (there's two of them)